ledger-service/http-json/src/it/scala/http/WebsocketServiceIntegrationTest.scala

```diff
@@ -450,7 +450,75 @@ class WebsocketServiceIntegrationTest
       } yield resumes.foldLeft(1 shouldBe 1)((_, a) => a)
   }
 
-  "fetch should should return an error if empty list of (templateId, key) pairs is passed" in withHttpService {
+  "fetch multiple keys should work" in withHttpService {
+    (uri, encoder, _) =>
+      def matches(expected: Seq[JsValue], actual: Seq[JsValue]): Boolean =
```

Comment:
As far as I can tell, the only difference between just using `==` (at least last I checked it was a bunch of case classes so equals does the right thing) is that you ignore additional keys in `actual` for objects. Is that intentional? If so why? If not, why don’t we use `==`?

Reply:
This is intentional. The goal is that the "expected" value is readable; there's a lot of noise otherwise.

Reply:
makes sense but probably worth adding a comment for this to make it clear that this is intentional.

Reply:
Comment added.